server/cluster.go

```diff
@@ -1098,6 +1110,34 @@ func (c *Cluster) gcProxySessionsForNode(node string) {
 	}
 }
 
+func (t *Topic) clusterSelectProxyEvent() (event ProxyEventType, s *Session, val *reflect.Value) {
```

Comment:
It looks like Go usually passes variables of the type `reflect.Value` by value rather than by reference.

Reply:
Yes, I was thinking about that as well.
I need to be able to return an empty value - hence I made it a pointer.